alicloud/table_alicloud_ram_policy.go

```diff
@@ -23,9 +23,11 @@ func tableAlicloudRamPolicy(_ context.Context) *plugin.Table {
 		Description:      "Alibaba Cloud RAM Policy",
 		DefaultTransform: transform.FromCamel(),
 		Get: &plugin.GetConfig{
-			KeyColumns:        plugin.AllColumns([]string{"policy_name", "policy_type"}),
-			ShouldIgnoreError: isNotFoundError([]string{"InvalidParameter.PolicyType", "EntityNotExist.Policy", "MissingParameter"}),
-			Hydrate:           getRAMPolicy,
+			KeyColumns: plugin.AllColumns([]string{"policy_name", "policy_type"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"InvalidParameter.PolicyType", "EntityNotExist.Policy", "MissingParameter"}),
```

Comment:
Do we need to add `EntityNotExist.Policy` error code here?
We have already handled this in [DefaultGetConfig](https://github.com/turbot/steampipe-plugin-alicloud/pull/344/files#diff-5f561e6fa39974710222f0ead49a126eec949864b6c87646c290c36cc88db682R17) for this plugin.